### ISSUE TITLE
Import declaration of type does works if an imported variable name does not match the original type name in the namespaces approach.

### ISSUE THREAD

**Issue by dimarudnev** [TypeStrong/typedoc#1400]
## Search terms
namespaces, import, renaming, types

## Steps to reproduce the bug

1. Create the d.ts file with the following content in the empty folder
```
declare module NamespaceA {
	export interface InterfaceA { }
	//export type renamedInterfaceA =  InterfaceA; // WA to typedoc v0.15
}
declare module NamespaceB {
	export interface InterfaceA { }
}
declare module NamespaceC {
	import renamedInterfaceA = NamespaceA.InterfaceA;
	import InterfaceA = NamespaceB.InterfaceA;
	    
	export class TestClass {
		testMethod1(options: renamedInterfaceA): any;
		testMethod2(options: InterfaceA): any;
	}
}

```
2. Run
```
typedoc --includeDeclarations --excludePrivate --target ES5 .
```
See the generated types of the parameter of testMethod1 and testMethod2
The JSON output equvalent: [typedoc019.ts405.zip](https://github.com/TypeStrong/typedoc/files/5583163/typedoc019.ts405.zip)

## Expected Behavior

The  "option" parameter of testMethod1 is referred to the NamespaceA.InterfaceA.
The  "option" parameter of testMethod2 is referred to the NamespaceB.InterfaceA.

## Actual Behavior

The "option" parameter of testMethod1 has the incorrect type and referred to the root of NamespaceC. In json renamedInterfaceA is variable inside NamespaceC without type.
The "option" parameter of testMethod2 has the correct type and referred to the NamespaceB.InterfaceA.

## Environment

-   Typedoc version: 0.19.2
-   TypeScript version: 4.0.5
-   Node.js version: 12.18.3
-   OS: windows 10

## Possible related tickets:

https://github.com/TypeStrong/typedoc/issues/1353
https://github.com/TypeStrong/typedoc/issues/435

Does this issue is going to be fixed in v0.20 ?

**Comment by Gerrit0** [comment]
Yep, this is fixed in v0.20. I had to make one change because I apparently broke global files, but in v0.20.0-beta.12 this produces the expected result.

**Comment by dimarudnev** [comment]
I have also tested my issue solution with typedoc@0.20.0-beta.14 and it works as expected.  Thank you, very much for your work. Typedoc is a very helpful tool!
